syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTListenAddress";
option optimize_for = LITE_RUNTIME;

import public "ContractEnums.proto";

message ListenAddress {
    optional uint32 version = 1;
    optional AddressType type = 2;
    optional ProtocolVersion protocol = 3;
    optional string host = 4;
    optional uint32 port = 5;
}
